src/jsMain/kotlin/io/beatmaps/user/account.kt

```diff
@@ -163,6 +173,65 @@ class AccountComponent : RComponent<AccountComponentProps, AccountComponentState
                     }
                 }
             }
+            div("mb-3") {
+                label("form-label") {
+                    attrs.htmlFor = "description"
+                    +"Description"
+                }
+                textarea(classes = "form-control") {
+                    key = "description"
+                    attrs.id = "description"
+                    attrs.value = state.description
+                    attrs.rows = "5"
+                    attrs.onChangeFunction = {
+                        setState {
+                            description = descriptionRef.current?.value ?: ""
+                        }
+                    }
+                    ref = descriptionRef
+                }
+                div("d-grid") {
+                    button(classes = "btn btn-success", type = ButtonType.submit) {
+                        attrs.onClickFunction = { ev ->
+                            ev.preventDefault()
+
+                            if (props.userDetail.description == state.description) {
+                                setState {
+                                    descriptionErrors = listOf("That's already your description!")
+                                    usernameErrors = listOf()
+                                }
+                            } else {
+                                setState {
+                                    userLoading = true
+                                }
+
+                                Axios.post<ActionResponse>(
+                                    "${Config.apibase}/users/description",
+                                    AccountDetailReq(state.description),
+                                    generateConfig<AccountDetailReq, ActionResponse>()
+                                ).then {
+                                    if (it.data.success) {
+                                        window.location.reload()
```

Comment:
Would probably be nicer to either call loadState in the parent (pass down a method as a property to trigger this) or allow this to directly modify the user state shown on the page.